api/routes/3DPrintingForm.js

```diff
@@ -45,7 +46,13 @@ router.post('/GetForm', (req, res) => {
 
   PrintingForm3D.find(obj, (error, forms) => {
     if (error) {
-      logger.log(`3DPrinting /GetForm error: ${error}`);
+      const info = {
+        userEmail: req.body.email,
+        errorTime: new Date(),
+        apiEndpoint: '3DPrintingForm/GetForm',
+        errorDescription: err
```

Comment:
you checked that this was a string?

Reply:
[https://docs.mongodb.com/stitch/mongodb/actions/collection.find/](Yes)